index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="navbar-innercontainer">
+            <span class="nav-logo">My Logo</span>
+            <ul class="nav-items">
+                <li class="nav-link">Portfolio</li>
+                <li class="nav-link">About</li>
+                <li class="nav-link">Contact</li>
+                <li class="nav-link"><img src="./Assets/logos/envelope.png" class="icon" alt="" /></li>
+            </ul>
+            <div class="baricon">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </div>
+        </div>
+    </nav>
+    <section class="herosection">
+        <div class="textcontainer">
+            <h5 class="h5">Hey There I’m Life.</h5>
+            <h6 class="h6">I'm a Software Developer</h6>
+            <p>
+                I can help you build a product , feature or website Look through some
+                of my work and experience! If you like what you see and have a project
+                you need coded, don’t hesitate to contact me.
+            </p>
+        </div>
+
+        <div class="icon-container">
```

Comment:
Good work creating a container for all the social icons, but there's something to improve here.

- As this is a list of items, the correct tag to use here instead a `div`, is the `ul` tag.

- Following this you can wrap each icon inside a `li` tag and each `li` inside an `a` tag with the corresponding `img`

It could be something like this structure

![image](https://user-images.githubusercontent.com/93630700/227517329-23d77aba-ccab-4e74-a297-cb24a87f6ff8.png)